specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/iotcentral.json

```diff
@@ -0,0 +1,1221 @@
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/privateEndpointName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This is returned as a response to the status polling request for the delete operation. The body contains the operation information that tracks the deletion status.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationStatusResult"
+            }
+          },
+          "202": {
+            "description": "Accepted - Delete request accepted; the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "Once the long running delete operation completes successfully, a 204 No Content status code is returned."
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.IoTCentral/iotApps/{resourceName}/privateEndpointConnections": {
+      "get": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Get all Private Endpoint Connections of a IoT Central Application.",
+        "operationId": "PrivateEndpoints_List",
+        "x-ms-examples": {
+          "PrivateEndpoints_List": {
+            "$ref": "./examples/PrivateEndpoints_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This is a synchronous operation. The body contains a JSON-serialized array of the metadata from all the Private Endpoint Connections of the IoT Central Application.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/privatelinks.json#/definitions/PrivateEndpointConnectionListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.IoTCentral/iotApps/{resourceName}/privateLinkResources": {
```

Comment:
you are missing an individual GET, section 7.5.3 in NRPs doc [https://microsoft.sharepoint.com/:w:/r/teams/vnet/Shared%20Documents/Integration%20Pattern%20Specs/Private%20Endpoints/%5BNRP%5D%20Private%20Endpoint%20Design.docx?d=w9d7659170bbb4f779883ab675b9edc26&csf=1&web=1&e=hY2tWS](https://microsoft.sharepoint.com/:w:/r/teams/vnet/Shared%20Documents/Integration%20Pattern%20Specs/Private%20Endpoints/%5BNRP%5D%20Private%20Endpoint%20Design.docx?d=w9d7659170bbb4f779883ab675b9edc26&csf=1&web=1&e=hY2tWS)

Reply:
thank you for pointing that out. i'll add it.